### PR TITLE
Update China mod to Annimarush suggested changes

### PR DESCRIPTION
**Pull request by EmilAminy**
Voted in with 28 votes to 5 (84.8% support)
For season 21

### CODE REVIEW

File: sweddit_combined/common/ideas/000_su_ideas.txt [hunk 1]
```diff
@@ -363,43 +350,41 @@ NNG_ideas = {
...
 	free = yes		#will be added at load.
 	
 	CHC_1 = {
-		manpower_recovery_speed = 0.1
-		global_manpower_modifier = 0.20
+		movement_speed = 0.1
 	}
 	CHC_2 = {
-		global_trade_goods_size_modifier = 0.1
+		prestige_decay = -0.1
```
Comment (ludlyl) [inline sweddit_combined/common/ideas/000_su_ideas.txt]: "2) Prestige Decay -1% / Max Promoted Cultures +1"

File: sweddit_combined/decisions/ChineseFormables.txt [hunk 2]
```diff
@@ -1,143 +1,76 @@
...
-	}
-	
 	form_chinese_empire = {
 		major = yes
 		potential = {
 			NOT = { has_country_flag = formed_chinese_empire }
 			normal_or_historical_nations = yes
-			was_never_end_game_tag_trigger = yes
+			OR = {
+				tag = MNG
```
Comment (ludlyl) [inline sweddit_combined/decisions/ChineseFormables.txt]: Why is ming here?

File: sweddit_combined/decisions/ChineseFormables.txt [hunk 3]
```diff
@@ -1,143 +1,76 @@
...
+				owns_core_province = 699 #Lanzhou
 			}
-			OR = {
-				south_china_region= {
-					type = all
-					owned_by = ROOT
-				}
-				north_china_region= {
-					type = all
+			calc_true_if = {
```
Comment (ludlyl) [inline sweddit_combined/decisions/ChineseFormables.txt]: > One of the following must be true
>   All provinces in the South China region:
>      Owned by X
>   All provinces in the North China region:
>      Owned by X

Why is this removed and exchanged for what is here now?

File: sweddit_combined/decisions/ChineseFormables.txt [hunk 4]
```diff
@@ -1,143 +1,76 @@
...
+				all_province = {
+					superregion = china_superregion
 					owned_by = ROOT
 				}
+				amount = 50
```
Comment (ludlyl) [inline sweddit_combined/decisions/ChineseFormables.txt]: Why is this here?

File: sweddit_combined/decisions/ChineseFormables.txt [hunk 5]
```diff
@@ -154,19 +87,46 @@ country_decisions = {
 				}
 				add_permanent_claim = IMP
 			}
-			add_prestige = 25
+			add_prestige = 50
```
Comment (ludlyl) [inline sweddit_combined/decisions/ChineseFormables.txt]: Reason behind changing this?

File: sweddit_combined/decisions/ChineseFormables.txt [hunk 6]
```diff
@@ -154,19 +87,46 @@ country_decisions = {
...
-			add_government_reform = imperial_examination
+			add_government_reform = chinese_empire_reform
 		}
 		ai_will_do = {
 			factor = 1
 		}
 		ai_importance = 400
 	}
+
+	adopt_confucian_administration = {
```
Comment (ludlyl) [inline sweddit_combined/decisions/ChineseFormables.txt]: Should normal conversion to confucian_state_reform really exist if this decision exists?

File: sweddit_combined/history/countries/CGS - Changsheng.txt [hunk 7]
```diff
@@ -1,5 +1,4 @@
 government = monarchy
-add_government_reform = chinese_warlord
```
Comment (ludlyl) [inline sweddit_combined/history/countries/CGS - Changsheng.txt]: Should changsheng (and similar tags) be left without a default reform instead of having confucian_state_reform to start with?

File: sweddit_combined/history/countries/CHC - Chu.txt [hunk 8]
```diff
@@ -1,7 +1,41 @@
...
+		female = no
+		regent = no
+		religion = confucianism
+		culture = hubei
+	}
+	clear_scripted_personalities = yes
+	add_ruler_personality = inspiring_leader_personality
+	add_ruler_personality = drunkard_personality
+	add_ruler_personality = architectural_visionary_personality
+}
```
Comment (ludlyl) [inline sweddit_combined/history/countries/CHC - Chu.txt]: Should these really be set? Not a big thing i guess but still wasn't specified.

File: sweddit_combined/history/countries/CHC - Chu.txt [hunk 9]
```diff
@@ -1,7 +1,41 @@
...
+		dynasty = Chu
+		birth_date = 1424.03.21
+		death_date = 1445.01.01
+		claim = 95
+		adm = 3
+		dip = 3
+		mil = 4
+	}
+	clear_scripted_personalities = yes
+	add_heir_personality = conqueror_personality
```
Comment (ludlyl) [inline sweddit_combined/history/countries/CHC - Chu.txt]: Same as above

File: sweddit_combined/history/countries/CSH - Shun.txt [hunk 10]
```diff
@@ -1,7 +1,7 @@
 government = monarchy
-add_government_reform = chinese_warlord
+add_government_reform = confucian_state_reform
```
Comment (ludlyl) [inline sweddit_combined/history/countries/CSH - Shun.txt]: Why is this reform set here but not on all other confucianism warlords?

File: sweddit_combined/history/countries/JIN - Jin.txt [hunk 11]
```diff
@@ -1,6 +1,37 @@
...
+		claim = 95
+		adm = 3
+		dip = 3
+		mil = 4
+	}
+	clear_scripted_personalities = yes
+	add_ruler_personality = martial_educator_personality
+	add_ruler_personality = lawgiver_personality
+	add_ruler_personality = well_connected_personality
+	add_heir_personality = tactical_genius_personality
```
Comment (ludlyl) [inline sweddit_combined/history/countries/JIN - Jin.txt]: Will all of these go to the monarch and none to the ruler?

And same as above, should these really be set.

File: sweddit_combined/history/countries/WUU - Wu.txt [hunk 12]
```diff
@@ -1,6 +1,36 @@
...
+		death_date = 1445.01.01
+		claim = 95
+		adm = 3
+		dip = 2
+		mil = 4
+	}
+	clear_scripted_personalities = yes
+	add_ruler_personality = tactical_genius_personality
+	add_ruler_personality = well_advised_personality
+	add_heir_personality = naive_personality
```
Comment (ludlyl) [inline sweddit_combined/history/countries/WUU - Wu.txt]: Same as above.

File: sweddit_combined/history/provinces/2165 - Pu'er.txt [hunk 13]
```diff
@@ -17,4 +17,6 @@ is_city = yes
 
 discovered_by = chinese
 discovered_by = nomad_group
-discovered_by = indian
\ No newline at end of file
+discovered_by = indian
+
+
```
Comment (ludlyl) [inline sweddit_combined/history/provinces/2165 - Pu'er.txt]: This (and similar) files that doesn't change anything shouldn't be pushed.

File: sweddit_combined/history/provinces/707 - Yumen.txt [hunk 14]
```diff
@@ -2,40 +2,18 @@
...
 base_production = 2
 base_manpower = 1
 is_city = yes
-extra_cost = 16
-center_of_trade = 2
-
 discovered_by = chinese
 discovered_by = nomad_group
+extra_cost = 16
+center_of_trade = 2
```
Comment (ludlyl) [inline sweddit_combined/history/provinces/707 - Yumen.txt]: Are these vanilla values?

File: sweddit_combined/localisation/em_modifers_l_english.yml [hunk 15]
```diff
@@ -1,4 +1,4 @@
-﻿l_english:
+l_english:
```
Comment (ludlyl) [inline sweddit_combined/localisation/em_modifers_l_english.yml]: This file shouldn't have been pushed.

Probably best to remove all of these and squash merge the PR.

File: sweddit_combined/localisation/pollit_l_english.yml [hunk 16]
```diff
@@ -1,4 +1,4 @@
-﻿l_english:
+l_english:
```
Comment (ludlyl) [inline sweddit_combined/localisation/pollit_l_english.yml]: Same as above

File: sweddit_combined/localisation/z_ages_l_english.yml [hunk 17]
```diff
@@ -1,2 +1,2 @@
-﻿l_english:
+l_english:
```
Comment (ludlyl) [inline sweddit_combined/localisation/z_ages_l_english.yml]: Same as above

File: sweddit_combined/events/warringchina.txt [hunk 18]
```diff
@@ -16,12 +16,6 @@ country_event = {
 	immediate = {
 		hidden_effect = {
 			set_country_flag = claims_on_china
-			ROOT = {
```
Comment (ludlyl) [inline sweddit_combined/events/warringchina.txt]: What is the effect of removing this? Will the claims still be set but without any pop-up?

File: sweddit_combined/common/government_reforms/99_sweddit_reforms.txt [hunk 19]
```diff
@@ -0,0 +1,133 @@
...
+	start_territory_to_estates = 0.5
+	potential = {
+		NOT = { tag = IMP }
+		culture_group = east_asian
+		religion = confucianism
+		OR = {
+			tag = CHC
+			tag = YUE
+			tag = JIN
+			tag = WUU
```
[the remark this answers is not in the document]
Comment (EmilAminy) [reply sweddit_combined/common/government_reforms/99_sweddit_reforms.txt]: Öhh, har faktiskt inte läst annimas text, jag har

[the remark this answers is not in the document]
Comment (EmilAminy) [reply sweddit_combined/common/government_reforms/99_sweddit_reforms.txt]: Öhh, jag har faktiskt inte läst annimas text, jag hade honom på disc och vi gjorde alla ändringar ihop. Jag vet inte om det va han eller jag som la in det där men jag antar att vi bara glömde bort vad som stod i förslaget. Kan fixa.

File: sweddit_combined/common/government_reforms/99_sweddit_reforms.txt [hunk 20]
```diff
@@ -0,0 +1,133 @@
...
+	legacy_equivalent = feudal_monarchy
+	start_territory_to_estates = 0.5
+	potential = {
+		religion = confucianism
+		NOT = {
+			tag = IMP
+			tag = CHC
+			tag = YUE
+			tag = JIN
+			tag = WUU
```
[the remark this answers is not in the document]
Comment (EmilAminy) [reply sweddit_combined/common/government_reforms/99_sweddit_reforms.txt]: Det här va bara för att det var fult att ha båda synliga samtidigt. Eftersom vi glömde devven i den andra så körde på motsatta conditionet här.

[the remark this answers is not in the document]
Comment (EmilAminy) [reply sweddit_combined/common/government_reforms/99_sweddit_reforms.txt]: Kan också fixa

File: sweddit_combined/common/ideas/000_su_ideas.txt (hunk 1 above)
Comment (EmilAminy) [reply sweddit_combined/common/ideas/000_su_ideas.txt]: Tror det här bara är ett misstag, fixar också

File: sweddit_combined/decisions/ChineseFormables.txt (hunk 2 above)
Comment (EmilAminy) [reply sweddit_combined/decisions/ChineseFormables.txt]: Ming är end game tag, så de får inte forma EoC, la in de så att de också kune

File: sweddit_combined/decisions/ChineseFormables.txt (hunk 3 above)
Comment (EmilAminy) [reply sweddit_combined/decisions/ChineseFormables.txt]: Mitt egna förslag för att undvika probelemet med att undvika problemet med Taiwan när man vill forma Kina. Spelmässigt ungefär lika svårt men lite mer dynamiskt. Kan ta bort.

File: sweddit_combined/history/provinces/707 - Yumen.txt (hunk 14 above)
Comment (EmilAminy) [reply sweddit_combined/history/provinces/707 - Yumen.txt]: Tror det, jag körde bara mitt script i vanilla-mappen och tog de filer jag fick ut

File: sweddit_combined/events/warringchina.txt (hunk 18 above)
Comment (EmilAminy) [reply sweddit_combined/events/warringchina.txt]: This is the claim on all of china event, and it triggers the generate leader event which has now been removed. Thought it might cause issues when we removed the generate leader event so I took out the trigger.

File: sweddit_combined/localisation/pollit_l_english.yml (hunk 16 above)
Comment (EmilAminy) [reply sweddit_combined/localisation/pollit_l_english.yml]: Misstag, tar och fixar

File: sweddit_combined/localisation/z_ages_l_english.yml (hunk 17 above)
Comment (EmilAminy) [reply sweddit_combined/localisation/z_ages_l_english.yml]: Misstag, tar och fixar

File: sweddit_combined/decisions/ChineseFormables.txt (hunk 4 above)
Comment (EmilAminy) [reply sweddit_combined/decisions/ChineseFormables.txt]: Same as the other one about taiwan

File: sweddit_combined/history/countries/CHC - Chu.txt (hunk 8 above)
Comment (EmilAminy) [reply sweddit_combined/history/countries/CHC - Chu.txt]: Annima says ruler traits were in the proposal

File: sweddit_combined/history/countries/CHC - Chu.txt (hunk 9 above)
Comment (EmilAminy) [reply sweddit_combined/history/countries/CHC - Chu.txt]: Same as above

File: sweddit_combined/history/countries/WUU - Wu.txt (hunk 12 above)
Comment (EmilAminy) [reply sweddit_combined/history/countries/WUU - Wu.txt]: Same as above

File: sweddit_combined/decisions/ChineseFormables.txt (hunk 5 above)
Comment (EmilAminy) [reply sweddit_combined/decisions/ChineseFormables.txt]: Den här skyller jag på annima, det är han som gjort. tar bort

File: sweddit_combined/history/countries/CGS - Changsheng.txt (hunk 7 above)
Comment (EmilAminy) [reply sweddit_combined/history/countries/CGS - Changsheng.txt]: The ones without reforms don't have cores in the beginning of the game. Only way for them to spawn is through rebels. If they still manage to appear they'll be able to pick whatever reform they want.

File: sweddit_combined/history/countries/CSH - Shun.txt (hunk 10 above)
Comment (EmilAminy) [reply sweddit_combined/history/countries/CSH - Shun.txt]: The ones that are set exist at start of game
